src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -492,6 +492,56 @@
         </Setter>
     </Style>
 
+    <Style x:Key="PackageManagerDownloadVersionButton"
+           TargetType="{x:Type Border}">
+        <Setter Property="Background" Value="#000000" />
+        <Setter Property="Margin" Value="3" />
+
+        <Style.Resources>
+            <Style TargetType="{x:Type Button}">
+                <Setter Property="Template">
+                    <Setter.Value>
+                        <ControlTemplate>
+
+                            <Border x:Name="container"
+                            Background="Transparent">
+                                <TextBlock x:Name="text"
+                                    HorizontalAlignment="Center"
+                                    VerticalAlignment="Center"
+                                    Margin="4,0"
+                                    FontSize="11"
+                                    Foreground="#BBB"
+                                    FontWeight="Bold"
+                                    Text="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=Content}" />
+                            </Border>
+
+                            <ControlTemplate.Triggers>
+
+                                <Trigger Property="Button.IsMouseOver"
+                                 Value="true">
+                                    <Setter TargetName="container"
+                                    Property="Background"
+                                    Value="#333333" />
+                                </Trigger>
+
+                                <Trigger Property="IsEnabled"
+                                 Value="false">
+                                    <Setter TargetName="text"
+                                    Property="Foreground"
+                                    Value="#666" />
+                                </Trigger>
+
+                            </ControlTemplate.Triggers>
+
+                        </ControlTemplate>
+                    </Setter.Value>
+                </Setter>
+            </Style>
+
+        </Style.Resources>
+
+    </Style>
+
```

Comment:
If this `Style` isn't used anywhere else than in `PackageManagerSearchView.xaml`, then there's no reason to add to the cost of start-up. Can we move this style into `PackageManagerSearchView.xaml` so it's only parsed once when that dialog is shown?